pkg/types/build_info.go

```diff
@@ -38,6 +38,17 @@ type ErrImageBuild struct {
 	Err   error
 }
 
+func NewBuildInfo(params Params, path string, transportMethod TransportMethod, startedAt time.Time) *BuildInfo {
+	bi := new(BuildInfo)
+	bi.Params = params
+	bi.Path = path
+	bi.TransportMethod = transportMethod
+	bi.StartedAt = startedAt
+	bi.ExitCode = 1
```

Comment:
I think we should use a special exit code value here to indicate the error condition that the container has not run and this exit code has not been returned by the container. So setting it to `1` doesn't allow us to determine whether this came from the container, or if this is a failure to even start the container

We already do this here
https://github.com/skroutz/mistry/blob/b02896475ff89e0e87f3798a0a477ce5325e7bd2/cmd/mistryd/worker.go#L282-L287

So I'm suggesting to initialize with `bi.ExitCode = -999`, and move the -999 value into a const

Reply:
Sounds good.